test/e2e/tests/comments/inline.spec.ts

```diff
@@ -16,96 +16,173 @@ interface InlineCommentCreationTestProps {
 // The base test for creating an inline comment.
 async function inlineCommentCreationTest(props: InlineCommentCreationTestProps) {
   const {page, createDraftDocument} = props
-
-  // 1. Create a new draft document
-  await createDraftDocument('/test/content/input-ci;commentsCI')
-
-  // 2. Click the overlay to active the editor.
-  await page.waitForSelector('[data-testid="activate-overlay"]', WAIT_OPTIONS)
-  await page.click('[data-testid="activate-overlay"]')
-
-  // 3. Select all text in the editor.
-  await page.waitForSelector('[data-testid="pt-editor"]', WAIT_OPTIONS)
-  const editor = page.locator('[data-testid="pt-editor"]')
-  await page.waitForSelector('div[role="textbox"][contenteditable=true]', WAIT_OPTIONS)
-  const textBox = editor.locator('div[role="textbox"][contenteditable=true]')
-  await textBox.selectText()
+  test.slow()
+
+  // Function to perform all steps up to text selection
+  async function setupAndSelectText() {
+    // 1. Create a new draft document
+    await createDraftDocument('/test/content/input-ci;commentsCI')
+
+    // Wait for network to become idle to ensure document is fully loaded
+    await page.waitForLoadState('load', {timeout: WAIT_OPTIONS.timeout * 2})
```

Comment:
I understand that the WAIT_OPTIONS and waiting for concept was already something in this test, but relying on load plus timeouts makes me nervous 😬